v-next/hardhat-keystore/src/internal/utils/get-keystore-file-path.ts

```diff
@@ -0,0 +1,15 @@
+import path from "node:path";
+
+import { ensureDir } from "@ignored/hardhat-vnext-utils/fs";
+import envPaths from "env-paths";
+
+export async function getKeystoreFilePath(): Promise<string> {
+  const configDirPath = await getConfigDir();
+  return path.join(configDirPath, "keystore.json");
+}
+
+async function getConfigDir(): Promise<string> {
```

Comment:
I suspect we shouldn't be handling this in the plugin directly.

Are we expecting Hardhat plugins to be able to write to the Hardhat config global folder?

## Options

1. Allow the direct writing to the folder using envPaths
2. Setup the global config path during hre initialation - should plugins know about this folder?
3. Provide it as a `hardhat-util` helper functions that can be used by Hardhat plugins

Reply:
Already got to this in a comment above. I think we should go with 3.